### ISSUE TITLE
Add descriptive aria-label to content report popover close button

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#1015]
## Description

The content report popover is missing an aria-label on the close button. This causes JAWS to read the entire popover contents (which is a lot of text). NVDA just doesn't say anything (I believe that'd be VoiceOver's behavior as well).

## Expectation

Add an `aria-label` with a descriptive value like `"Close content report menu"`

Also: make `aria-label` a required prop on the `VIconButton` component; there's no reason it shouldn't be.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.
